practice-app/src/main/helpers/issue_helper.py

```diff
@@ -29,3 +35,111 @@ def get_issue(json_issue):
         ret_issue['labels'] = json_issue['labels']
 
     return ret_issue
+
+
+DB_LOC = "C:/GIT/2021SpringGroup12/practice-app/sqlfiles/practice-app.db"
+
+
+def insert_issue(cur: Cursor, issue: Issue):
+    cur.execute(
+        "INSERT OR IGNORE INTO Issues(number, description, state) VALUES (?,?,?)",
+        (issue.number, issue.description, issue.state))
+    for assignee in issue.assignees:
+        cur.execute(
+            "INSERT OR IGNORE INTO Assignees(issue_number, assignee) VALUES (?,?)",
+            (issue.number, assignee))
+    for label in issue.labels:
+        cur.execute(
+            "INSERT OR IGNORE INTO Labels(issue_number, label) VALUES (?,?)",
+            (issue.number, label))
+
+
+def insert_single_issue(issue: Issue):
```

Comment:
maybe you can delete this function and send single issue as `list(issue)` to `insert_multiple_issue()`. But just a suggestion, not necessary.

Reply:
It makes so much sense